stdlib/2and3/distutils/version.pyi

```diff
@@ -0,0 +1,35 @@
+# Stubs for distutils.version (Python 2 and 3.5)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+import sys
+from typing import Any
+
+class Version:
+    def __init__(self, vstring=None): ...
+
+    if sys.version_info >= (3,):
```

Comment:
It looks odd that you have to declare these elementary operations (`__eq__` etc.), but only for Python 3.